app/views/main/_summary.erb

```diff
@@ -0,0 +1,25 @@
+<div class='section'>
+  <% if @grade_entry_forms.empty? && @assignments.empty? %>
+      <p><%= t(:no_assignments_and_grade_entry_forms_message) %></p>
+  <% else %>
+    <div class="table">
+      <table>
+        <thead>
+        <tr>
+          <th><%= t(:name) %></th>
+        </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>
+              <%= link_to 'Course Summary',
+                          view_summary_course_summary_path('course_summary'),
```

Comment:
You aren't using path helpers properly. Read about them at http://guides.rubyonrails.org/routing.html. Related: it doesn't make sense to have `id: 'summary'` in the `data` on the next line.